DHL/Datatype/GB/ExportLineItem.php

```diff
@@ -86,7 +86,7 @@ class ExportLineItem extends Base
             'required' => false,
             'subobject' => false,
             'comment' => 'Commodity codes for shipment type',
-            'minLength' => '1',
+            'minLength' => '0',
```

Comment:
I'm wary of this change, as it seems to be trying to make this optional... but we will leave it in here for now. I'd love to know the justification for why this was needed, though.